Gemfile

```diff
@@ -84,6 +89,8 @@ group :development, :test do
   gem 'guard-rspec', require: false
 
   gem 'factory_bot_rails'
+  # TODO: switch to a version number once that version is released
```

Comment:
I don't understand why any of the changes in the Gemfile were made.

Reply:
Currently in master if you locally run `bundle exec rspec` you will get multiple

```
RuntimeError:
  can't modify frozen Array
# ./config/environment.rb:5:in `<top (required)>'
# ./spec/rails_helper.rb:5:in `<top (required)>'
# ./spec/serializers/institution_serializer_spec.rb:3:in `<top (required)>'
```

Additionally this is the same dependency pulled in the same way as in https://github.com/department-of-veterans-affairs/vets-api/blob/master/Gemfile#L141

In https://github.com/department-of-veterans-affairs/gibct-data-service/pull/426 it was recommended to remove this line, removal didn't affect build passing but it affects local rspec

Reply:
`sass-rails` and `rainbow` are only needed as gems when the development and test groups are not installed

Reply:
I this case I don't think you want to copy-paste from vets-api.  The comment `switch to a version number once that version is released` is out of place, considering the version containing this commit was released 2 years ago.

Reply:
So because of https://thoughtbot.com/blog/deprecating-static-attributes-in-factory_bot-4-11 the version will be set to `4.10.0` which is the version listed on the previously listed commit hash https://github.com/thoughtbot/factory_bot/commit/50eeb67241ea78a6b138eea694a2a25413052f49

An attempt to fix the issue referenced was not going to be simple, another story will be needed to update factory_bot